src/_gettsim/parameters/unterhaltsvors.yaml

```diff
@@ -30,25 +30,91 @@ altersgrenze_mindesteinkommen:
   2017-01-01:
     scalar: 12
     reference: Art. 23 G. v. 14.08.2017 BGBl. I S. 3122.
-altersgrenzen:
+altersgrenzen_bezug:
   name:
     de: Altersgrenzen für den Bezug von Unterhaltsvorschuss
     en: age limits for receiving alimony payments
   description:
     de: >-
       Kinder, die das 12. Lebensjahr noch nicht vollendet haben und bei einem
       alleinerziehenden Elternteil leben haben Anspruch auf Unterhaltszahlungen. Zudem
-      können auch Kinder bis zur Vollendung des 18. Lebensjahres Unterhaltsvorschuss
-      bekommen, wenn das Elternteil ein Mindesteinkommen hat.
+      können seit Juli 2017 auch Kinder bis zur Vollendung des 18. Lebensjahres
+      Unterhaltsvorschuss bekommen, wenn das Elternteil ein Mindesteinkommen hat.
     en: >-
       Children under the age of 12 living with a single parent are entitled to alimony
-      payments. In addition, children up to the age of 18 receive advance on alimony
-      payments if the parent has income beyond a minimal threshold.
+      payments. In addition, since July 2017, children up to the age of 18 receive
+      advance on alimony payments if the parent has income beyond a minimal threshold.
   reference: § 1 Abs. 1, 1a UhVorschG
-  2017-01-01:
-    1: 6
-    2: 12
-    3: 18
+  2008-01-01:
```

Comment:
Convert to a list, as above. And does it make sense to just add the numbers (betrag, potentially the factor 0.87) in here?